Languages/English/Keyed/Keys.xml

```diff
@@ -79,6 +79,7 @@
   <CE_FireModes>Fire modes</CE_FireModes>
   <CE_AimedBurstCount>Burst shot count</CE_AimedBurstCount>
   <CE_UsedBy>Used by</CE_UsedBy>
+  <CE_CookingOff>Cooking off {0}/{1} bullets</CE_CookingOff>
```

Comment:
This seems very gamey and not like something the player ought to have any way of knowing.